package.json

```diff
@@ -20,7 +20,8 @@
   "dependencies": {
     "colors": "^1.4.0",
     "minimist": "^1.2.6",
-    "sax": "^1.2.4"
+    "sax": "^1.2.4",
+    "odata-abnf": "github:oasis-tcs/odata-abnf"
```

Comment:
This belongs into devDependencies: we are _not_ building an EDMX-Validator, we are _testing_ our XSD files.